PelotonEppSdkTests/ClientAuthTokenTests.cs

```diff
@@ -0,0 +1,217 @@
+            Assert.AreEqual(null, getResult.AuthorizedDatetime);
+            Assert.AreEqual(null, getResult.CreditCardToken);
+            Assert.AreEqual(null, getResult.BankAccountToken);
+            Assert.AreEqual(null, getResult.TransactionRefCode);
+            Assert.AreEqual(null, getResult.EventToken);
+        }
+
+        [TestMethod]
+        public void TestGetClientAuthTokenExpired()
+        {
+            // expired: 5a1c7be3998a4205a2b1c8c49a0e4599
+            var cat = "5a1c7be3998a4205a2b1c8c49a0e4599";
+            var getRequest = GetBasicRequest(_clientId, _clientKey, "PelonEppSdkTests");
+
+            getRequest.ReturnUrl = null;
+            getRequest.ClientAuthToken = cat;
+            var getResult = getRequest.GetAsync().Result;
+            Debug.WriteLine(getResult.Message);
+            Debug.WriteLineIf((getResult.Errors != null && getResult.Errors.Count >= 1), string.Join("; ", getResult.Errors ?? new List<string>()));
+            Assert.IsTrue(getResult.Success);
+            Assert.AreEqual(0, getResult.MessageCode);
+            Assert.AreEqual("Success", getResult.Message);
+            Assert.AreEqual(32, getResult.ClientAuthToken.Length);
+            Assert.IsTrue(getResult.Errors == null || !getResult.Errors.Any());
+            Assert.AreEqual("", getResult.ReturnUrl);
+            Assert.IsFalse(getResult.Active);
+            Assert.AreEqual(null, getResult.AuthorizedDatetime);
+            Assert.AreEqual(null, getResult.CreditCardToken);
+            Assert.AreEqual(null, getResult.BankAccountToken);
+            Assert.AreEqual(null, getResult.TransactionRefCode);
+            Assert.AreEqual(null, getResult.EventToken);
+        }
+
+
+        [TestMethod]
+        public void TestGetClientAuthTokenAuthorizedCreditCardToken()
+        {
+            // authorized: 8c0a15527850422ea505b9bbda076eb4
+            var cat = "8c0a15527850422ea505b9bbda076eb4";
+            var getRequest = GetBasicRequest(_clientId, _clientKey, "PelonEppSdkTests");
+
+            getRequest.ReturnUrl = null;
+            getRequest.ClientAuthToken = cat;
+            var getResult = getRequest.GetAsync().Result;
+            Debug.WriteLine(getResult.Message);
+            Debug.WriteLineIf((getResult.Errors != null && getResult.Errors.Count >= 1), string.Join("; ", getResult.Errors ?? new List<string>()));
+            Assert.IsTrue(getResult.Success);
+            Assert.AreEqual(0, getResult.MessageCode);
+            Assert.AreEqual("Success", getResult.Message);
+            Assert.AreEqual(32, getResult.ClientAuthToken.Length);
+            Assert.IsTrue(getResult.Errors == null || !getResult.Errors.Any());
+            Assert.AreEqual("", getResult.ReturnUrl);
+            Assert.IsFalse(getResult.Active);
+            Assert.AreEqual(ClientAuthTokenAuthorizationType.CreditCard, getResult.Type);
+            Assert.AreEqual("2017-02-27 21:30:58", getResult.AuthorizedDatetime.ToString());
```

Comment:
I think this test will always fail or will fail at some point if you are checking a static date.